netmanager/src/views/pages/Roles/index.js

```
@@ -27,36 +27,32 @@
-          );
-        });
-    }
+    if (!activeNetwork) return;
+    getNetworkPermissionsApi(activeNetwork._id)
+      .then((res) => {
+        setPermissions(res.permissions);
+      })
+      .catch((error) => {
+        dispatch(
+          updateMainAlert({
+            message: error.response && error.response.data && error.response.data.message,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)